src/extension.ts

```diff
@@ -547,15 +546,17 @@ export async function activate(context: vscode.ExtensionContext) {
 			const uris: vscode.Uri[] = [];
 
 			for (const job of Object.values(state.job.entities)) {
-				if (
-					!job ||
-					!doesJobAddNewFile(job.kind) ||
-					job.newContentUri === null
-				) {
+				if (!job) {
 					continue;
 				}
 
-				uris.push(vscode.Uri.parse(job.newContentUri));
+				if (job.newContentUri !== null) {
+					uris.push(vscode.Uri.parse(job.newContentUri));
+				}
+
+				if (job.oldContentUri !== null) {
```

Comment:
I think this `oldContentUri` is the same as `oldUri` right now, because we don't preformat files right now.